ecl/dedicated_hypervisor/v1/vcenter.py

```diff
@@ -68,3 +71,19 @@ def update(self, session, vcenter_id, password, license_id):
         )
         self._translate_response(resp, has_body=True)
         return self
+
+
+class Link_local_addresses(VCenter):
+    base_path = '/vcenters/addresses'
```

Comment:
```suggestion
    base_path = '/vcenters/addresses'
    resources_key = "addresses"
    resource_key = "address"
```
resources_keyが定義されていません。
今回は List APIしか使いませんが、念のため resource_keyも定義しておきますか。
resources_keyがなんのために必要なのか、杉本の作ったノウハウ資料で理解しておいてください。

Reply:
ご指摘のとおり修正します。